pkgs/development/tools/ocaml/ocaml-lsp/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, ocamlPackages }:
+
+with ocamlPackages; buildDunePackage rec {
```

Comment:
Why do you use `buildDunePackage` (as you seem to customize the build and install phases nonetheless)?

Reply:
no special reason, at first I tried to use `buildDunePackage` with the default phases, but it didn't work. 
Should I change it to `stdenv.mkDerivation`?

Reply:
Please use what you think is more appropriate.